src/psyclone/psyir/nodes/operation.py

```diff
@@ -220,6 +223,18 @@ class BinaryOperation(Operation):
        :returns: the value of the upper bound of the `index` dimension of \
                  `array`.
 
+    Casting Operators:
+
+    .. function:: REAL(arg0, precision)
+
+       :returns: `arg0` converted to a floating point number in the \
```

Comment:
Would "of" instead of "in" be better? Please ignore if this is just my non-native English and both work well here.

Reply:
Yes, I think you're right. Done (and for INT too).